cmd/v/help/new.txt

```diff
@@ -1,5 +1,13 @@
 Usage:
-  v new [name] [description]
-  v init
+  v new [flags] [name] [description]
 
-Setup file structure for a V project.
+Examples:
+  v new
+  v new -d
+  v new MyProject
```

Comment:
Don't we need an example on how to pass a description here too?
```
v new MyProject "My Description"
```

Reply:
I could add it, but examples are meant as a cut-out and they don't have to show all available options

Reply:
Please add it, not everyone knows that quoting is needed in shells to treat something as a single argument, and descriptions usually are larger than a single word, so it will be a common case.